lib/crypto/secp256k1/secp256k1.go

```diff
@@ -147,6 +151,17 @@ func (k *PublicKey) Verify(msg, sig []byte) (bool, error) {
 	return secp256k1.VerifySignature(k.Encode(), msg, sig), nil
 }
 
+// UnmarshalPubkey converts bytes to a secp256k1 public key.
```

Comment:
does this take an uncompressed pubkey (65 bytes)? could you update comment to specify format of the expected input